app/src/main/java/cm/aptoide/pt/download/DownloadAnalytics.java

```diff
@@ -282,36 +290,37 @@ private void setUpInstallEvent(String md5, String packageName, String trustedVal
           navigationTracker.getCurrentScreen(), md5, packageName, trustedValue, action,
           previousContext,
           offerResponseStatus.equals(WalletAdsOfferManager.OfferResponseStatus.ADS_HIDE),
-          isMigration);
+          isMigration, hasAppc);
     } else {
       downloadCompleteEvent(navigationTracker.getPreviousScreen(),
           navigationTracker.getCurrentScreen(), md5, packageName, trustedValue, action,
-          previousContext, isMigration);
+          previousContext, isMigration, hasAppc);
     }
   }
 
   public void installClicked(String md5, String packageName, AnalyticsManager.Action action,
-      WalletAdsOfferManager.OfferResponseStatus offerResponseStatus) {
+      WalletAdsOfferManager.OfferResponseStatus offerResponseStatus, boolean hasAppc) {
     String previousContext = navigationTracker.getViewName(false);
 
     if (!offerResponseStatus.equals(WalletAdsOfferManager.OfferResponseStatus.NO_ADS)) {
 
       downloadCompleteEvent(navigationTracker.getPreviousScreen(),
           navigationTracker.getCurrentScreen(), md5, packageName, null, action, previousContext,
-          offerResponseStatus.equals(WalletAdsOfferManager.OfferResponseStatus.ADS_HIDE), false);
+          offerResponseStatus.equals(WalletAdsOfferManager.OfferResponseStatus.ADS_HIDE), false,
+          hasAppc);
     } else {
```

Comment:
This method is being called from SeeMoreAppcManager and it is sending the download complete event with isMigration as false.